venter/tests.py

```diff
@@ -246,27 +250,25 @@ def test_send_mass_mail(self):
         request = SimpleNamespace()
 
         auth_mail = Authorities.objects.create(email='receiver1@example.com', name='receiver')
-
         # Reported Complaint
-        complaints = Complaints.objects.create(created_by=self.user.profile, status=STATUS_REPORTED,
-                                               description='Test Complaint', authority_email=auth_mail)
+        complaint_1 = Complaints.objects.create(created_by=self.user.profile, status=STATUS_REPORTED,
+                                                description='Test Complaint')
+        complaint_1.authorities.add(auth_mail)
 
         # In Progress Complaint with images
-        Complaints.objects.create(created_by=self.user.profile, status=STATUS_IN_PROGRESS, authority_email=auth_mail)
+        complaint_2 = Complaints.objects.create(created_by=self.user.profile, status=STATUS_IN_PROGRESS)
+        complaint_2.authorities.add(auth_mail)
         image = []
-        image.append(ComplaintMedia.objects.create(image_url='https://www.google.com/', complaint=complaints))
-        complaints.images.set(image)
+        image.append(ComplaintMedia.objects.create(image_url='https://www.google.com/', complaint=complaint_1))
+        complaint_2.images.set(image)
 
-        # Complaint with no authority
-        Complaints.objects.create(created_by=self.user.profile, status=STATUS_REPORTED)
```

Comment:
This is an important test case. Why remove it?

Reply:
Reinstated